src/components/organisms/EventModal/EventModal.scss

```diff
@@ -12,7 +11,7 @@ $EventModal--padding-top: 40px;
     padding: $EventModal--padding-top $spacing--xl $spacing--xl;
     display: flex;
     flex-direction: column;
-    background-color: $dark-grey;
+    background-color: $background--modal;
```

Comment:
NOTE: just unifying the modal background color under a single constant named for its purpose